tg/views/__init__.py

```diff
@@ -249,6 +251,35 @@ def resize(self, rows: int, cols: int, p: float = 0.5) -> None:
         self.win.resize(self.h, self.w)
         self.win.mvwin(0, self.x)
 
+    def _get_flags(self, msg_proxy: MsgProxy):
+        flags = []
+        chat = self.model.chats.chats[self.model.current_chat]
+        my_id = self.users.get_me()["id"]
+        states = {
+            "messageSendingStateFailed": "failed",
+            "messageSendingStatePending": "pending",
+        }
+        if (
+            msg_proxy.sender_id != my_id
```

Comment:
you could use `if self.model.is_me(msg_proxy.sender_id)` like [here](https://github.com/paul-nameless/tg/pull/45/files#diff-5c4c547a58bbeca9b9c7a58ca0f8b87aR426)